Purchases/FoundationExtensions/DateFormatter+Extensions.swift

```diff
@@ -14,32 +14,85 @@
 
 import Foundation
 
-extension DateFormatter {
+/// A type that can convert from and to `Dates`.
+public protocol DateFormatterType {
```

Comment:
does the protocol need to be public?

Reply:
I think I made it public for a reason, but maybe not, good point! Let me check.

Reply:
Good catch! Just changed it to `internal` :)